content/zh/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -176,25 +176,25 @@ Kubernetes 会为命名端口创建 SRV 记录，这些端口是普通服务或
 其中包含 Pod 端口号和形为 `auto-generated-name.my-svc.my-namespace.svc.cluster-domain.example`
 的域名。
 
-## Pods
+## Pods {#pods}
```

Comment:
If it's not appeared in en content, I think it may be unnecessary.

Reply:
removed it.